designs/administrator/4.4/quay-imagemanifestvuln-list/index.md

```diff
@@ -0,0 +1,29 @@
+---
+parent: Administrator
+version: 4.4
+---
+
+# Quay Image Manifest Vulnerabilities List
+
+Users that have installed the Container Security Operator now have direct access to known container image vulnerabilities in their cluster via the new **Image Manifest Vulnerabilities** nav item in the Administration section. This nav item lists the vulnerabilities on the cluster and each can be viewed to see specific details and the pods that are affected by it. Users can also access this list filtered by pod from a pod's actions menu.
+
+## Image Manifest Vulnerabilities List and Details
+
+![ImageManifestVulnerabilities List](img/1imvlist.png)
+- A new nav item  **Image Manifest Vulnerabilities**  is added to the Administration section.
+- An brief overview of the vulnerability is viewable in the list view with a link to the Quay instance the image is hosted on (for example, Quay.io.)
```

Comment:
The way I'm reading this bullet as a newbie, clicking a manifest ID (like `35c1c5688e7`) takes you to the URL-based location that the image is hosted on. I don't know much about manifests, but would a separate column like "Location" or "Hosted by" with external links that say "quay.io" or "quay3.mycompany.io" make more sense than linking to them from the manifest string? 🤷‍♂️